### DISCUSSION TITLE
How to configure for perpetual towns that never expire?

### DISCUSSION THREAD

**Discussion by GarretSidzaka** [TownyAdvanced/Towny#7074] [Q&A] [answered]
My server is very lowkey and only consensual pvp/war allowed.   My players are worried that if they are inactive for a long time, their towns will be destroyed.   Is there a way to make towns perpetual and never be destroyed?

**Comment by LlmDl** [comment, marked as answer]
Make your town upkeep costs 0, and set the old_resident_delete feature to false in the config.

**Reply by GarretSidzaka** [reply]
i found these:
delete_old_residents

price_nation_upkeep: '0.0'
    price_town_upkeep: '0.0'

**Reply by LlmDl** [reply]
You've figured it out.
